### PR TITLE
Named axis support

### PR DESCRIPTION
**Pull request by thierryH91200**

### CODE REVIEW

**Comment by thierryH91200** [comment]
capture d ecran 2018-03-21 10 24 58
i have a little problem with the legend
something has changed ??

capture d ecran 2018-03-21 10 24 42
capture d ecran 2018-03-21 10 17 32

**Comment by jjatie** [comment]
Please fix CI issues.

File: ChartsDemo-iOS/ChartsDemo-iOS.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -658,11 +658,13 @@
...
 					225B361C1F6EB9A50005B3D5 = {
 						CreatedOnToolsVersion = 9.0;
+						ProvisioningStyle = Manual;
 					};
 					5B57BBAE1A9B26AA0036A6CC = {
 						CreatedOnToolsVersion = 6.1.1;
+						DevelopmentTeam = 84WLNJ2YG4;
 						LastSwiftMigration = 0900;
-						ProvisioningStyle = Manual;
+						ProvisioningStyle = Automatic;
```
Comment (jjatie) [inline ChartsDemo-iOS/ChartsDemo-iOS.xcodeproj/project.pbxproj]: Please revert the provisioning preferences.

**Comment by jjatie** [comment]
Can be reopened with Conflicts and change requests resolved.

**Pull request closed without merging**